.github/workflows/verify-this-repo.yml

```diff
@@ -0,0 +1,10 @@
+name: Verify Pull Request
```

Comment:
Why do we need this here?

Reply:
What do you mean?

Reply:
Why do we need this workflow verifying PRs in this repo? This isn't an OSOCE submodule in the sense that OSOCE provides any build support (it would be different if the OSOCE build would use the GitHub-Actions submodule's workflows without branch specifiers in YML).

Reply:
Fair enough.